schema/deploy/seed_todo.sql

```diff
@@ -0,0 +1,11 @@
+-- Deploy todo_app:seed_todo to pg
+-- requires: insert_todo
+-- requires: todos
+-- requires: todo_appschema
+
+BEGIN;
+
+select todo_app.insert_todo('task not done', false);
+select todo_app.insert_todo('task done', true);
+
+COMMIT;
```

Comment:
for seed data, we've been following the following pattern in our project:

- if the data is actually "seed" data, i.e. the initial data which can later be modified by our users, then a sqitch migration, like this is good
- if the data is hardcoded data, which users are not expected to change, but developers may want to change between releases, we use a different set of sql scripts containing idempotent changes ([example from the ciip repo](https://github.com/bcgov/cas-ciip-portal/blob/658842b59d7f778dc86ba59d8963edd5baebf901/schema/data/prod/gas.sql)).